src/components/grid-view/GridView.scss

```diff
@@ -2,32 +2,19 @@
 @import '../../styles/variables';
 
 .bdl-GridView {
-    padding: 0 5px;
```

Comment:
This was not working as expected.  This div appears entirely to the left of the actual grid view content, so in reality, it was adding 10px of padding to the left of the content, and 0 px of padding to the right of the content.